server/run.py

```diff
@@ -38,7 +38,7 @@ def main(mode=None):
 
     configure_logging(log_path=config.log)
     init_db(db=config.db, enable_logging=False)
-    fill_db()
+    #fill_db()
```

Comment:
Are you really wont to submit this?

Reply:
new run option was added to create configuration